### DISCUSSION TITLE
Administrator account and videos disappear

### DISCUSSION THREAD

**Discussion by zarevskaya** [mediacms-io/mediacms#1071] [Q&A]
**Describe the issue**
Hello, yesterday everything was working fine, this morning I want to log in with my administrator account and it is refused. And at the same time, my videos are no longer present.

I specify that I sent my videos with the administrator account.

This is the second time this has happened, randomly. If I pull docker-compose again, the admin account registered in docker-compose no longer has any effect. I just have a normal account. I was even able to create the same login and password and account name without problem. Without being an admin.

**Environment (please complete the following information):**
 - OS: Debian 12
 - Installation method: Docker
 
I don't have any particular logs, unfortunately.

Have a nice day!

**Comment by mgogoulos** [comment]
Hi, what has happened to the installations since yesterday, was it running or you have shut it down? It looks as if the DB data was removed, and in this case everything was re-created. The DB files are found in ../postgres_data .

**Reply by zarevskaya** [reply]
Thank you for your reply. Nothing has been changed, I went to sleep leaving everything as is. I have not changed anything.

Both times this happened, it always started with a password rejection, which was the correct one.

**Comment by zarevskaya** [comment]
What's also strange is that by doing docer-compose up -d, the administrator account no longer works.
